src/appliance-onboarding-script/appliance_setup/avs/avsarconboarder/orchestrator/network/_network_orchestrator.py

```diff
@@ -0,0 +1,57 @@
+from ...constants import Constant
+from ...orchestrator._orchestrator import Orchestrator
+import ipaddress
+from avs.avsarconboarder.exception import InvalidInputError
+
+class NetworkOrchestrator(Orchestrator):
+
+    def validate_ip_address(self, ip_addr):
+        try:
+            ipaddress.ip_address(ip_addr)
+        except:
+            raise InvalidInputError("Invalid ip addresses provided")
+
+    def validate_static_ip_cidr_block(self, segmant_ip_cidr, version):
```

Comment:
NIT: segment_ip_cidr and not segmant_ip_cidr

Reply:
Same for a lot of other places in the PR

Reply:
made changes